include/zephyr/drivers/dali.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2025 by Sven Hädrich <sven.haedrich@sevenlab.de>
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
There has been an update to the [recommended copyright notice](https://docs.zephyrproject.org/latest/contribute/guidelines.html#copyrights-notices):

```suggestion
/*
 * Copyright The Zephyr Project Contributors
 * Copyright Sven Hädrich <sven.haedrich@sevenlab.de>
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
There is nothing wrong with the copyright statement already in this PR. Please see https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects for further details.